airflow/www_rbac/templates/airflow/chart.html

```diff
@@ -0,0 +1,57 @@
+{#
```

Comment:
Same with templates -- are these the same between www and www_rbac? If so can we use the www/templates/ instead of copying?

Reply:
The templates are not quite the same between www and www_rbac:
- they extends from different master templates (/appbuilder rather than /admin) and imports different html/jinja macros
- some hard-coded urls in the html files had to be adjusted for FAB's endpoints

There are some files that no longer apply for FAB, like query.html/chart.html. I'll take some time and remove the old css/js/html files